src/agents/DexPoolScannerAgent.ts

```diff
@@ -0,0 +1,30 @@
+import {injectable} from 'inversify';
+
+import {LoopAgent} from './LoopAgent.js';
+import {BlockchainScanner} from 'src/services/BlockchainScanner.js';
+
+@injectable()
+export abstract class DexPoolScannerAgent extends LoopAgent {
+  abstract backoffTime: number;
+  abstract interval: number;
+  abstract stepInterval: number;
+  abstract dexPoolScanner: BlockchainScanner;
+
+  async execute(): Promise<void> {
+    const result = await this.dexPoolScanner.run();
+
+    if (!result.success && !result.waitBlockRange) {
+      this.logger.error('[EthereumUniswapV3PoolScannerAgent] Pool scan failed. Waiting...');
```

Comment:
```suggestion
      this.logger.error('${logPrefix} Pool scan failed. Waiting...');
```

same in other places